src/components/List.js

```diff
@@ -107,33 +155,37 @@ const List = ({ token }) => {
       {errorMessage && <Error errorMessage={errorMessage} />}
 
       <ul style={{ listStyleType: 'none' }}>
-        {docs &&
-          docs
-            ?.filter((doc) =>
-              doc?.itemName
+        {sortedList &&
```

Comment:
Now we are using the `const sortedList` that we assigned back in our sort function which removes the console warning we were seeing previously.